### PR TITLE
Add Scotto36 handwired keyboard

### PR DESCRIPTION
**Pull request by joe-scotto**
## Description

Add Scotto36 handwired keyboard along with `default` and `scotto` keymaps.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/handwired/scottokeebs/scotto36/info.json [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        // 4, 5, 6, 7, 8, 9, A3, A2, A1, A0
+        "cols": ["D4", "C6", "D7", "E6", "B4", "B5", "F4", "F5", "F6", "F7"],
+        // 15, 14, 16, 10
+        "rows": ["B1", "B3", "B2", "B6"]
+    },
+    "url": "",
```
Comment (filterpaper) [inline keyboards/handwired/scottokeebs/scotto36/info.json:22]:
```suggestion
```

File: keyboards/handwired/scottokeebs/scotto36/readme.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Scotto36
+
+![Scotto36](https://i.imgur.com/MCGv0ZHh.jpeg)
+
+A 36 key handwired ortholinear ergo keyboard. Featuring an OLED display with Bongo Cat. Case files available [here](https://github.com/joe-scotto/scottokeebs).
+
+-   Keyboard Maintainer: [Joe Scotto](https://github.com/joe-scotto)
+-   Hardware Supported: ATmega32U4
+-   Hardware Availability: [Amazon](https://amazon.com)
```
Comment (filterpaper) [inline keyboards/handwired/scottokeebs/scotto36/readme.md:9]:
```suggestion
*   Keyboard Maintainer: [Joe Scotto](https://github.com/joe-scotto)
*   Hardware Supported: ATmega32U4
*   Hardware Availability: [Amazon](https://amazon.com)
```

You can _proactively_ reformat the readme file in your other PRs according to the [prescribed template](https://github.com/qmk/qmk_firmware/blob/master/data/templates/keyboard/readme.md).

File: keyboards/handwired/scottokeebs/scotto36/readme.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+Flashing example for this keyboard:
+
+    make handwired/scottokeebs/scotto36:default:flash
+
+# Bootloader
+
+Uses [bootmagic](https://github.com/qmk/qmk_firmware/blob/master/docs/feature_bootmagic.md) allowing you to hold the top left key (0, 0) when plugging the board in to enter bootloader mode.
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```
Comment (filterpaper) [inline keyboards/handwired/scottokeebs/scotto36/readme.md:25]:
```suggestion
See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).

# Bootloader

Uses [bootmagic](https://github.com/qmk/qmk_firmware/blob/master/docs/feature_bootmagic.md) allowing you to hold the top left key (0, 0) when plugging the board in to enter bootloader mode.
```

**Review by filterpaper — CHANGES_REQUESTED** [review]

File: keyboards/handwired/scottokeebs/scotto36/readme.md (hunk 2 above)
Comment (joe-scotto) [reply keyboards/handwired/scottokeebs/scotto36/readme.md]: @filterpaper Done, updated them to use the template.

**Review by dunk2k — APPROVED** [review]

**Review by dunk2k — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Review by waffle87 — APPROVED** [review]

**Pull request merged**